app/decorators/article_decorator.rb

```diff
@@ -26,26 +24,29 @@ def url
   end
 
   def title_length_classification
-    if article.title.size > 105
+    if title.size > 105
       "longest"
-    elsif article.title.size > 80
+    elsif title.size > 80
       "longer"
-    elsif article.title.size > 60
+    elsif title.size > 60
       "long"
-    elsif article.title.size > 22
+    elsif title.size > 22
       "medium"
     else
       "short"
     end
   end
 
   def internal_utm_params(place = "additional_box")
+    org_slug = organization&.slug
```

Comment:
Good call!

Reply:
Is it possible for `organization&.slug` to be `nil`? Looking at this method, seems like we could be using a `nil` value for `org_slug` if `boosted_additional_articles` is truthy, right? Which means we could end up with `" _boosted"`.

I think this is how the current code also behaves, but just wondering if that's really the right thing to do here.

Reply:
I agree with you but I tried as much as possible not to change anything in the current behavior as this PR should be seamsless. Given we're talking about UTM params which are probably indexed by the Google bot, I also thought it wise not to change that, I'm unsure of the ramifications of changing a string there :)